src/caffeine/caffeine_h_m.f90

```diff
@@ -149,13 +171,13 @@ logical(c_bool) pure function caf_same_cfi_type(a, b) bind(C)
        import c_bool
        type(*), intent(in) :: a(..), b(..)
      end function
-  
+
      logical(c_bool) pure function caf_numeric_type(a) bind(C)
        !! bool caf_numeric_type(CFI_cdesc_t* a_desc);
        import c_bool
        type(*), intent(in) :: a(..)
      end function
-  
+
```

Comment:
These types of whitespace changes in lines that are not otherwise modified by a PR have the potential to slow down review and create spurious merge conflicts. Streamlined teamwork is far more important than a few stray/meaningless whitespace characters.

I assume these whitespace changes were performed automatically by your editor or some script. I highly encourage both @everythingfunctional and @ktras to DISABLE any such features that automatically create spurious whitespace changes over any files you happen to edit, to avoid these problems. I would even go so far as adding this to the coding standards; as an example, here are some of the relevant portions of the [GASNet coding standard](https://bitbucket.org/berkeleylab/gasnet/src/stable/README-devel):

```
* Do NOT use tab characters in GASNet code files. Use spaces to ensure correct
  formatting in any editor.
    Emacs users should: (setq-default indent-tabs-mode nil)
    Vim users should: :set expandtab

* Do NOT commit gratuitous changes to whitespace, even to remove tab characters.
  Such changes have no semantic content and can make use of revision history more
  difficult.  You can preview your changes with the following two commands:
     git diff -u | wc -l
     git diff -u -b | wc -l
  and if the outputs differ, then your changes include lines that have changed
  in only the whitespace characters.

* Large code motion (more than a dozen lines mostly unchanged) should be
  isolated in a commit by itself, whenever practical.

...
```

Reply:
Apologies for the accidental whitespace changes. I've undone them.